workflow/metrics/scripts/metrics/__init__.py

```diff
@@ -4,12 +4,10 @@
 from .isolated_labels import isolated_label_asw, isolated_label_f1, isolated_label_asw_y
 from .lisi import clisi, clisi_y, ilisi, ilisi_y
 from .nmi import nmi, nmi_kmeans_y, nmi_leiden_y
-from .pcr import cell_cycle, pcr, pcr_y
+from .pcr import cell_cycle, pcr, pcr_y, pcr_batch, pcr_label, pcr_random
 from .kbet import kbet_y
 from .morans_i import morans_i_batch, morans_i_label, morans_i_random
-from .morans_i_gene_scores import *
-from .principle_component_regression import pcr_batch, pcr_label, pcr_random
-
+from .gene_scores import *
```

Comment:
Could you do explicit imports here? This is recommended and makes it clearer which function comes from where as well as avoiding naming conflicts of imports and functions. (I admit that I probably broke that rule in the first place for a quick-and-dirty version, but now that we're finalising the code we should use the recommended way)